lightly/api/swagger_rest_client.py

```diff
@@ -61,16 +94,26 @@ def request(
             _request_timeout = self.timeout
 
         # Call RESTClientObject.request
-        return super().request(
-            method=method,
-            url=url,
-            query_params=query_params,
-            headers=headers,
-            body=body,
-            post_params=post_params,
-            _preload_content=_preload_content,
-            _request_timeout=_request_timeout,
-        )
+        try:
+            return super().request(
+                method=method,
+                url=url,
+                query_params=query_params,
+                headers=headers,
+                body=body,
+                post_params=post_params,
+                _preload_content=_preload_content,
+                _request_timeout=_request_timeout,
+            )
+        except ApiException as e:
+            if e.reason in [
+                ApiErrorCode.BAD_REQUEST,
+                ApiErrorCode.CONFLICT,
+                ApiErrorCode.MALFORMED_RESPONSE,
+                ApiErrorCode.MALFORMED_REQUEST,
+            ]:
+                raise e
+            raise PrettyPrintApiException(e).with_traceback(None) from None
```

Comment:
Has `with_traceback` an additional benefit over `from None`?

Reply:
No, it seems to not change anything, I removed it.